Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -211,13 +211,13 @@ theorem coeff_coe (i : ℤ) :
       not_false_iff]
 #align power_series.coeff_coe PowerSeries.coeff_coe
 
--- Porting note: simp can prove this, and removed norm_cast attribute
+-- Porting note (#10618): simp can prove this, and removed norm_cast attribute
```

Comment:
"and removed norm_cast attribute" seems to be an independent thing which we might also want to track?

Reply:
@jcommelin Yes, I'll integrate it as soon as I come back home this evening.